### PR TITLE
Global variables implementation

### PR DESCRIPTION
**Pull request by lucaseduoli**
This PR is related to the Global Variables functionality.

image
When clicking the Globe, it's possible to use global variables in any InputComponent.
When hovering an item, it's possible to delete it.
When clicking on Add New Variable, it's possible to add a new global variable.

### CODE REVIEW

File: langflow1.db
Comment (ogabrielluiz) [inline langflow1.db:1]: This should be removed

File: src/frontend/src/components/codeTabsComponent/index.tsx [hunk 1]
```diff
@@ -374,6 +383,28 @@ export default function CodeTabsComponent({
                                                         ].data.node.template[
                                                           templateField
                                                         ].value = target;
+                                                        if (
+                                                          globalVariablesEntries.includes(
+                                                            target
+                                                          )
+                                                        ) {
+                                                          setNoticeData({
+                                                            title: `the value inserted in ${templateField} is a global variable, \n
```
Comment (ogabrielluiz) [inline src/frontend/src/components/codeTabsComponent/index.tsx:392]: This text could be improved

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]

File: langflow1.db
Comment (lucaseduoli) [reply langflow1.db:1]: Resolved

File: src/frontend/src/components/codeTabsComponent/index.tsx (hunk 1 above)
Comment (lucaseduoli) [reply src/frontend/src/components/codeTabsComponent/index.tsx]: Resolved

**Review by ogabrielluiz — APPROVED** [review]
LGTM

**Review by anovazzi1 — CHANGES_REQUESTED** [review]
Variable value and mark lost after update component code, need to clean the variable field or keep the variable as it was

**Review by anovazzi1 — APPROVED** [review]
LGTM

**Pull request merged**